<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SunilBss" failures="2" tests="6" name="FirefoxTest" time="67.873" errors="0" timestamp="2024-10-23T15:45:07 IST">
  <testcase classname="testCases.TC002_LoginTest" name="LoginTest" time="1.728"/>
  <testcase classname="testCases.Login_DDT" name="verify_login" time="2.332"/>
  <testcase classname="testCases.Login_DDT" name="verify_login" time="12.67"/>
  <testcase classname="testCases.Login_DDT" name="verify_login" time="11.616">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.Login_DDT.verify_login(Login_DDT.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
  <testcase classname="testCases.Login_DDT" name="verify_login" time="11.653"/>
  <testcase classname="testCases.Login_DDT" name="verify_login" time="11.357">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.Login_DDT.verify_login(Login_DDT.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- FirefoxTest -->
